tensorly/decomposition/candecomp_parafac.py

```diff
@@ -148,6 +147,11 @@ def parafac(tensor, rank, n_iter_max=100, init='svd', svd='numpy_svd', tol=1e-8,
         Activate return of iteration errors
     non_negative : bool, optional
         Perform non_negative PARAFAC. See :func:`non_negative_parafac`.
+    mask : ndarray
+        array of booleans with the same shape as ``tensor`` should be 0 where
+        the values are missing and 1 everywhere else. Note:  if tensor is
+        sparse, then mask should also be sparse with a fill value of 1 (or
+        True).
```

Comment:
FYI this will require some extra work if you ever support a sparse library that doesn't support nonzero fill values the way that `sparse` does.